packages/core/saved-objects/core-saved-objects-server/src/saved_objects_index_pattern.ts

```diff
@@ -6,4 +6,13 @@
  * Side Public License, v 1.
  */
 
+/**
+ * Collect and centralize the names of the different saved object indices.
+ * Note that all of them start with the '.kibana' prefix.
+ * There are multiple places in the code that these indices have the form .kibana*.
+ * However, beware that there are some system indices that have the same prefix
+ * but are NOT used to store saved objects, e.g.: .kibana_security_session_1
+ */
 export const MAIN_SAVED_OBJECT_INDEX = '.kibana';
+export const TASK_MANAGER_SAVED_OBJECT_INDEX = `${MAIN_SAVED_OBJECT_INDEX}_task_manager`;
+export const SavedObjectsIndexPatterns = [MAIN_SAVED_OBJECT_INDEX, TASK_MANAGER_SAVED_OBJECT_INDEX];
```

Comment:
2 NITS:
1. case inconsistency between the individual indices constants and the list one
2. not a big fan of the `SavedObjectsIndexPatterns` naming. I think something like `ALL_SAVED_OBJECT_INDICES` would be more explicit

Reply:
We can do this as a follow up, to avoid merge conflict nightmares with the rest of "feature" PRs.